servicetalk-transport-netty-internal/src/main/java/io/servicetalk/transport/netty/internal/WireLoggingInitializer.java

```diff
@@ -34,11 +37,14 @@
      * @param loggerName The name of the logger to log wire events.
      */
     public WireLoggingInitializer(final String loggerName) {
-        loggingHandler = new LoggingHandler(loggerName, TRACE);
+        LogLevel logLevel = getNettyLogLevel(loggerName);
+        loggingHandler = logLevel != null ? new LoggingHandler(loggerName, logLevel) : null;
```

Comment:
Just a suggestion; feel free to ignore.

This approach precludes dynamic log level manipulation for existing channels. I think we should always add the handler if user opts-in to it using the builder setting.

Reply:
`logLevel  == null` will only if the logging level can't be mapped to a Netty `LogLevel` (e.g. "OFF"). The user's logging config can set the logging level set to a valid level (e.g. not "OFF") and use a filter to achieve the dynamic enable/disable of logs on pre-existing channels.